src/components/implementation/tests/unit_constant_value/unit_constant_value.c

```diff
@@ -3,14 +3,12 @@
 int
 main(void)
 {
-	if ( NUM_CPU == 4 )
-	{
+	if (NUM_CPU == 4){
```

Comment:
Still: space between ){. just look at most of the other code in the repo to see how this should be formatted.

Reply:
sorry for the oversight. I have added the space between ) and {

Reply:
Gosh, I miswrote here. We have spaces between ){ but not newlines.

This is why I said to just look at other code and emulate what we have.